docs/adr/adr-global-state-management.md

```diff
@@ -0,0 +1,167 @@
+# ADR: Use Jotai for Global State Management
```

Comment:
@fhenrich33 could you add something on the reactivevariable of Apollo too in here. We tried it out for a few small feature and would love to know how it compares with jotai.